_docs/life_of_intern.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Life of an Intern at Puppet"
+author: dishakareer
+categories:
+  - Week 1 Update 
+  - Week 2 Update
+  - Week 3 Update 
+  - Week 4 Update 
+tags:
+ - intern
+---
+
+![DishasBlogPicture](images/mypicture.jpg)
+
+
+## Welcome! 
+
+## Life of an Intern at Puppet
+
+Hello Everyone!
+
+Welcome to my Blog Page **"Life of an Intern at Puppet"**
+
+I am sure you cannot wait to hear about all the exciting things Puppet offers for their interns!!
+
+Why not start with Who Am I and how did I got into Puppet for my placement year?
+
+I am Disha Kareer, originally from India but living in UK from last 15 years, long time isn't?
+
+I started off my academics with Knockbreda High School. This is where I completed my GCSE's and then decided to
+study for my A-Levels in Newtownbreda High School. I am very hardworking, diligent and caring individual who always had a keen interest in the field of Computing and IT. I developed a strong interest in this career path since the age of 10. This was because I always enjoyed developing and designing things
```

Comment:
Where possible, please use [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) - they make for _much_ easier code reviews for documentation and help see exactly what changes to suggest and where. 😊 

```suggestion
I started off my academics with Knockbreda High School.
This is where I completed my GCSE's and then decided to study for my A-Levels in Newtownbreda High School.
I am very hardworking, diligent and caring individual who always had a keen interest in the field of Computing and IT.
I developed a strong interest in this career path since the age of 10.
This was because I always enjoyed developing and designing things.
```